src/main/java/org/opentripplanner/routing/api/request/refactor/request/NewRouteRequest.java

```diff
@@ -0,0 +1,308 @@
+package org.opentripplanner.routing.api.request.refactor.request;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Locale;
+import org.opentripplanner.model.GenericLocation;
+import org.opentripplanner.model.plan.SortOrder;
+import org.opentripplanner.model.plan.pagecursor.PageCursor;
+import org.opentripplanner.model.plan.pagecursor.PageType;
+import org.opentripplanner.routing.api.request.RequestModes;
+import org.opentripplanner.routing.api.request.RoutingRequest;
+import org.opentripplanner.routing.api.request.StreetMode;
+import org.opentripplanner.routing.api.request.refactor.preference.RoutingPreferences;
+import org.opentripplanner.routing.core.TraverseMode;
+import org.opentripplanner.routing.core.TraverseModeSet;
+import org.opentripplanner.routing.vehicle_rental.RentalVehicleType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// TODO: 2022-08-18 rename class when done
+public class NewRouteRequest {
```

Comment:
There is no name collision here, the old is called RoutingRequest.

Reply:
I'm gonna rename it RoutingRequest when I'm done with the refactoring. It's just easier to refactor if the classes have different names